pkg/katautils/config.go

```diff
@@ -994,6 +993,19 @@ func SetKernelParams(runtimeConfig *oci.RuntimeConfig) error {
 		}
 	}
 
+	// set the scsi scan mode to none for virtio-scsi
+	if runtimeConfig.HypervisorConfig.BlockDeviceDriver == config.VirtioSCSI {
+		if !strings.Contains(formatted, "scsi_mod.scan") {
+			p := vc.Param{
+				Key:   "scsi_mod.scan",
+				Value: "none",
```

Comment:
thanks, can you update the unit test `TestSetKernelParams`?

Reply:
Also, can you change the title of this PR as well.

Reply:
ok